planetarium/NineChronicles
# Fix bug in adding main stat.

### Description

1. Fixes bug in displaying main stat with adding sub option values.
2. Prevent NRE thrown in `SkillView`.

### How to test

1. Open item tooltip with skills and check if NRE never happens.
2. Check if main stat of equipment is displaying properly.

### Related Links
https://app.asana.com/0/958521740385861/1200703786388842

### Required Reviewers

@Namyujeong @planetarium/9c-dev
